src/BloomExe/BloomExe.csproj

```diff
@@ -115,6 +115,10 @@
       <HintPath>..\..\packages\MarkdownDeep.NET.1.5\lib\.NetFramework 3.5\MarkdownDeep.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="MarkdownSharp" Condition="'$(OS)'!='Windows_NT'">
```

Comment:
I doubt that comments are preserved in csproj files, so I didn't bother putting one in.  The Squirrel assembly provides the MarkdownSharp class for Windows, but Linux packaging cannot tolerate the Squirrel assembly because it references some windows specific assemblies.  But our code uses the MarkdownSharp class so we have to provide it somehow for use on Linux.  I'll add a comment to the csproj file, but if visual studio throws it away someday, I probably won't notice.  :-)